modules/tiled-gallery.php

```diff
@@ -30,22 +30,3 @@ function jetpack_tiled_gallery_configuration_load() {
 }
 
 jetpack_load_tiled_gallery();
-
```

Comment:
Instead of completely deleting this, how about keeping it there but using the constant for `load_assets_as_required` as well, so it loads assets when present only?